resources/lib/SUBUtilities.py

```diff
@@ -10,6 +10,12 @@
 import xbmcvfs
 import xbmcaddon
 
+from unicodedata import normalize
```

Comment:
did you checked this plugin in Kodi 19? Cause sagi worked hard to make it work on it.

Reply:
Basically I'm still working only with kodi 17.6
But I can ask other users to test it if needed.
I guess it will work with kodi 19.

And as I said, this PR only relates to the Offline Subtitles Checking Support.

Reply:
Pls check it on Kodi 19, cause even wrong import will throw exception (check what Sagi have changed in his PR).

Reply:
@sagiben 
Sagi, can you test this PR for me with Kodi 19 (python 3)?

@CaTzil 
I have understood that you have Kodi 18 (python 2.7 & 3), so tell me if you can test it with your version too.

Thanks.

Those was added for this PR:
from unicodedata import normalize
from os import path
from requests import get
from json import loads, load
from time import time

Reply:
Hi, What exactly is the test case ?
I used plugin version 1.2.0 w/o the PR a TVShow with an Hebrew filename and was able to find subtitles for that tvshows automatically

Reply:
@sagiben 
Basically at first to check if using the addon throwing an exception due to wrong or unsupported imports, and second to test what you have already checked but in Kodi Library that scrap tv shows with Hebrew titltes (It's strange a little that it works for you even without the PR, maybe in Kodi 19 it's different)

But it seems that it's ok according to your comment. Thanks.